### DISCUSSION TITLE
Click on links in deleted messages (Twitch)

### DISCUSSION THREAD

**Discussion by ey34** [night/betterttv#5898] [Ideas]
I don't know if it is been said before. It would be really nice to be able to click on links in deleted messages. Copy-pasting is sometimes very tiring.

**Comment by night** [comment]
Thanks for the suggestion. We used to keep them linked, but after Twitch requested some number of years ago we changed them to be unlinked to improve safety against malicious links. Because of the circumstances of why this was changed, we do not have any current plans to revert this behavior.

**Reply by ey34** [reply]
What if you just made the twitch clips linked?

**Reply by night** [reply]
Unfortunately as this was an ask from the platform we have no intentions to revert any behavior at this time.
